acceptance_tests/features/search_respondent_by_email.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Internal user can search for a respondent via email
+  As an internal user
+  I need to be able to search for a registered respondent via their email
+  So that I can easily find them when dealing with operational calls
+
+
+  @us091_s01
+  Scenario: Search verified respondent by email address
+    Given the respondent account with email example@example.com has been created
```

Comment:
We could template the email address to pass it into the step as a parameter with double quotes, saving having to replace it in multiple places in the future.